pastis/launchers/run_luvoir_efields.py

```diff
@@ -1,58 +1,71 @@
 import os
 from astropy.io import fits
 import numpy as np
+
 from pastis.config import CONFIG_PASTIS
 import pastis.util as util
 from pastis.matrix_generation.matrix_from_efields import MatrixEfieldLuvoirA
-from pastis.pastis_analysis import  calculate_segment_constraints
-from pastis.simulators.luvoir_imaging import LuvoirA_APLC
+from pastis.pastis_analysis import calculate_segment_constraints
 import pastis.plotting as ppl
 
+
 if __name__ == '__main__':
+
+    NUM_RINGS = 1
```

Comment:
"NUM_RINGS" is no where called in this script. This launcher is for LuvoirA_APLC, I don't think this variable is required here.

Reply:
Ah yes, will delete.